### PR TITLE
Route1 car value

### PR DESCRIPTION
**Pull request by AndyGuffey**
Added API functionality all passing unit tests- plz review :)

### CODE REVIEW

File: routes/route1.js
Comment (DhanishaHarshad) [inline routes/route1.js:1]: great comments and easy to understand code!

Comment (DhanishaHarshad) [reply routes/route1.js:1]: error messages are clear to the user what the issue is

File: tests/route1.test.js [hunk 1]
```diff
@@ -148,19 +153,24 @@ describe("Car Value Suggestion", () => {
   //TODO ========================
   //TODO ==     Edge Cases     ==
   //TODO ========================
-  // describe("Edge Case Scenarios", () => {
```
Comment (DhanishaHarshad) [inline tests/route1.test.js]: assuming this is an older test that is commented out, have not reviewed

File: routes/route1.js [hunk 2]
```diff
@@ -1,8 +1,61 @@
...
+      const char = model[i].toUpperCase();
+      // Check if it is a letter (A-Z)
+      if (char >= "A" && char <= "Z") {
+        // Convert to position (A=1, B=2, ... Z=26)
+        const position = char.charCodeAt(0) - 64;
+        letterSum += position;
+      }
+      // Igonore spaces and non-letter characters
+    }
+  }
```
Comment (DhanishaHarshad) [inline routes/route1.js:53]: I dont really understand what is happening here but it looks good! Looks like it will be easy to maintain and update

**Review by DhanishaHarshad — COMMENTED** [review]
Every thing looks really tidy! There are some esctions that look like they could be combined for DRY princibles - lines 15 to 22 in the R2 file more specifically. 

line 45 - should this be a $valuue with the right syntax? I didnt see a test for this so it might be why it was not picked up

**Review by DhanishaHarshad — COMMENTED** [review]
Other thn that im happy to approve this!!!

File: routes/route1.js [hunk 3]
```diff
@@ -1,8 +1,61 @@
...
 });
 
+router.post("/", (req, res) => {
+  // Get model and year from the request body
+  const { model, year } = req.body;
+  // -----------------------
+  // -- Validation Checks --
+  // -----------------------
+
+  // Check if model is empty string
```
Comment (karlmacababbad) [inline routes/route1.js:16]: Validation points make sense but could be refactored due to multiple consequtive if statements.
